src/types/PoolOperation.sol

```diff
@@ -0,0 +1,26 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.24;
+
+import {PoolKey} from "../types/PoolKey.sol";
+import {BalanceDelta} from "../types/BalanceDelta.sol";
+
+/// @notice Parameter struct for `ModifyLiquidity` pool operations
+struct ModifyLiquidityParams {
```

Comment:
I see you removed the interface, can we confirm that this still solves the issue? When I implemented this without the interface, it didn't fix the circular dependency

Reply:
Yes, this does still solve the issue. Once these structs are moved out of IPoolManager, the circular references between IHooks and IPoolManager are eliminated and the monorepo builds cleanly.